src/DiffEq/MixMassFractionBetaCoeffPolicy.h

```diff
@@ -186,7 +186,7 @@ class MixMassFracBetaCoeffDecay {
         tk::real v = tk::ctr::lookup( tk::ctr::variance(depvar,c), moments );
 
         if (m<1.0e-8 || m>1.0-1.0e-8) m = 0.5;
-        if (v<1.0e-8 && v>1.0-1.0e-8) v = 0.5;
+        if (v<1.0e-8 || v>1.0-1.0e-8) v = 0.5;
```

Comment:
This seems like a semantic change? Is this a bug fix snook in?

Also given the numbers of a `-` in, i.e. `1e-8` do you think it's better to leverage spacing here:

` if (v<1.0e-8 || v>1.0-1.0e-8) v = 0.5;`

vs

` if ( v < 1.0e-8 || v > 1.0 - 1.0e-8 ) v = 0.5;`

Finally, as ever, I love braces {} in if's

Reply:
`if ( v<1.0e-8 || v>1.0 - 1.0e-8 ) v = 0.5;` 

may even be better middle ground